x-pack/docs/en/security/authentication/built-in-users.asciidoc

```diff
@@ -7,7 +7,27 @@ up and running. These users have a fixed set of privileges and cannot be
 authenticated until their passwords have been set. The `elastic` user can be
 used to <<set-built-in-user-passwords,set all of the built-in user passwords>>.
 
+.Create users with minimum privileges
+****
+The built-in users serve specific purposes and are not intended for general
+use. In particular, do not use the `elastic` superuser unless full access to
+the cluster is absolutely required. Instead, create users that have the minimum 
```

Comment:
```suggestion
the cluster is absolutely required. Instead, use it to create users that have the minimum 
```

Reply:
Should we also specify "on self-managed deployments"?

Reply:
@bytebilly, we could change this to read:

>On self-managed deployments, use the `elastic` user to create users that have the minimum necessary roles or privileges for their activities.

Is the `elastic` user not available (or not created) on Elastic Cloud deployments?

Reply:
suggestion works for me. On Cloud we still have `elastic`, but given there is single sign-on it's not the recommended way to authenticate for interactive users.